README.md

```diff
@@ -354,6 +315,49 @@ checkpointing so the job restarts near where it was interrupted.
     python3 xpk.py workload list \
     --cluster xpk-test --filter-by-job=$USER
     ```
+## Pathways on XPK
+
+* Cluster Create for Pathways:
+    Pathways compatible cluster can be created using `--enable-pathways`
+    ```shell
+    python3 xpk.py cluster create \
+    --cluster xpk-pw-test \
+    --num-slices=4 --on-demand \
+    --tpu-type=v5litepod-16 \
+    --enable-pathways
+    ```
+
+*   Workload Create for Pathways:
+    Pathways workload can be submitted using `--use-pathways` on a Pathways enabled cluster (created with `--enable-pathways`)
+    
+    Pathways workload example:
+    ```shell
+    python3 xpk.py workload create \
+    --workload xpk-pw-test \
+    --num-slices=1 \
+    --tpu-type=v5litepod-16 \
+    --cluster xpk-pw-test \
+    --use-pathways \
+    --server-image=<Pathways server image> \
+    --proxy-server-image=<Pathways proxy server image> \
+    --docker-name='user-workload' \
+    --docker-image=<maxtext docker image> \
```

Comment:
--base-docker-image flow is more recommended for xpk users so that local changes can be iterated on. Is that possible for pathways?

Reply:
Currently, the docker-image provided by the users is a fixed docker image, but if the others deps (mentioned) are merged, I can update docker-image to base-docker-image! I would also prefer that!